README.md

```diff
@@ -31,47 +32,81 @@
-   * Tutorial
-   * Migration guide from `Prelude`
-   * Haddock with examples for (almost) every function
-     (all examples are tested with [`doctest`](http://hackage.haskell.org/package/doctest))
-   * Documentation regarding [internal module structure]((http://hackage.haskell.org/package/relude/docs/Relude.html))
-   * `relude`-specific [HLint](http://hackage.haskell.org/package/hlint) rules:
-     [`.hlint.yaml`](.hlint.yaml)
+   + Tutorial
+   + Migration guide from `Prelude`
+   + Haddock for every function with examples tested by [`doctest`](http://hackage.haskell.org/package/doctest)
+   + Documentation regarding [internal module structure](http://hackage.haskell.org/package/relude/docs/Relude.html)
+   + `relude`-specific [HLint](http://hackage.haskell.org/package/hlint) rules: [`.hlint.yaml`](.hlint.yaml)
 7. **User-friendliness.** Ability to quickly migrate to `relude` if you're familiar
    with the common libraries like `text` and `containers`.
 8. **Exploration.** Experiment with new ideas and proposals without introducing
    breaking changes. `relude` uses the approach with `Extra.*` modules which are not
    exported by default so it's quite easy to bring something new and let users
    decide to use it or not.
 
-This README contains introduction to `Relude` and a tutorial on how to use it.
+This README contains introduction to `relude` and a tutorial on how to use it.
 
 ## Structure of this tutorial
 
 This tutorial has several parts:
 
-1. [Get started](#get-started-)
+1. [When to use an alternative prelude?](#when-to-use-an-alternative-prelude-)
+2. [Get started](#get-started-)
     * [base-noprelude](#base-noprelude-)
     * [Mixins](#mixins-)
     * [NoImplicitPrelude](#NoImplicitPrelude-)
-2. [Difference from Prelude](#difference-from-prelude-)
-3. [Reexports](#reexports-)
-4. [What's new?](#whats-new-)
+3. [Difference from Prelude](#difference-from-prelude-)
+4. [Reexports](#reexports-)
+5. [What's new?](#whats-new-)
 6. [Migration guide](#migration-guide-)
 7. [Comparison with other alternative preludes](#comparison-with-other-alternative-preludes-)
     * [Relude vs Protolude](#relude-vs-protolude-)
 8. [For developers](#for-developers-)
 
 This is neither a tutorial on _Haskell_ nor tutorial on each function contained
-in `Relude`. For detailed documentation of every function together with examples
+in `relude`. For detailed documentation of every function together with examples
 and usage, see [_Haddock documentation_](http://hackage.haskell.org/package/relude).
 
+## When to use an alternative prelude? [↑](#structure-of-this-tutorial)
+
+Module with the name `Prelude` is a module imported by default in every Haskell
+source file of your project. If you want to use some data types or functions
+which are not exposed by `Prelude`, you need to import them, adding necessary
+libraries to your project dependencies. Unlike ordinary libraries, alternative
+preludes replace default functions and data types.
```

Comment:
"Replace functions and data types" could sound a bit scary when interpreted incorrectly... Probably somehow rephrase this?